scripts/generate_dump

```diff
@@ -1241,6 +1259,12 @@ main() {
     end_t=$(date +%s%3N)
     echo "[ Capture Proc State ] : $(($end_t-$start_t)) msec" >> $TECHSUPPORT_TIME_INFO
 
+    
+    local asic="$(/usr/local/bin/sonic-cfggen -y /etc/sonic/sonic_version.yml -v asic_type)"
+    if [ "$asic" = "barefoot" ]; then
```

Comment:
Please move it next to "collect_broadcom"

Reply:
It is impossible because if to invoke collect_barefoot near collect_broadcom function then script does not make copy bf_drivers*.log files from /var/log into result zipped files